results-viewer/view.js

```diff
@@ -90,47 +103,58 @@ const view = (params, state, send) => {
 
   return choo.view`
     <div>
-    <dl>
-      <dt>Total Correctness:</dt>
-      <dd>${correctness.toFixed(3)}</dd>
-      <dt>Total Completeness:</dt>
-      <dd>${completeness.toFixed(3)}</dd>
-    </dl>
     <button onclick=${() => send('app:sort', { key: 'correctness_score:descending' })}>Most Correct</button>
     <button onclick=${() => send('app:sort', { key: 'correctness_score:ascending' })}>Least Correct</button>
     <button onclick=${() => send('app:sort', { key: 'completeness_score:descending' })}>Most Complete</button>
     <button onclick=${() => send('app:sort', { key: 'completeness_score:ascending' })}>Least Complete</button>
+    <button onclick=${() => send('app:sort', { key: 'index' })}>Reset</button>
     ${missingAccessToken}
     <ul class=${map ? 'sidebar' : ''}>
         <li class="header">
-          <div>Input Image</div>
-          <div>OSM "ground truth"</div>
-          <div>Net Prediction</div>
+          <div style=${colStyle}>Input Image</div>
+          <div style=${colStyle}>OSM "ground truth"</div>
+          ${results.map((result, i) => {
+            return choo.view`
+               <div style=${colStyle}>
+                 ${resultNames[i]}<br>
+                 <div class='metrics'>
+                   <nobr>Correct: ${result.metrics.correctness.toFixed(3)}</nobr><br>
+                   <nobr>Complete: ${result.metrics.completeness.toFixed(3)}</nobr><br>
+                 </div>
+               </div>`
+          })}
         </li>
         ${items
         .slice(0, state.app.limit)
         .map(item => {
           var tile = getTile(item)
           var image = tile ? getSatelliteTileURL(query, tile[0], tile[1], tile[2])
-            : (baseurl + item.input)
+            : (baseurls[0] + item.input)
```

Comment:
Oh yeah, so this is here for cases where the input data is not a standard map tile (so that we can't just use getSatelliteTileURL to construct a url based on x/y/z coords).  In that case, `baseurls[0] + item.input` would grab the "xxx_input.png" image that sits alongside "xxx_groundtruth.png" and "xxx_prediction.png".

The only case where we've used the latter is for the DG SpaceNet thing, where the images weren't tiled.

(Note that, as of https://github.com/developmentseed/skynet-internal/issues/3, those `xxx_input.png` images wouldn't be accessible on our ds-skynet working bucket; we'd need copy them to them to skynet-results to view.)